### PR TITLE
Tutorials reorg

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: docs/tutorials/mint-etherscan.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ title: Minting Tokens via Etherscan
...
-
-You can already mint tokens via the [CLI tool](tutorials/cli-tool.md), but you may prefer to interact with the contract more directly via Etherscan. All you’ll need to get started is some DAI to use as collateral (50 DAI should be enough) and ETH to pay for transaction gas (0.05 ETH should be enough).
+This article will walk you through minting tokens (like ETHBTC or yCOMP) for the first time on Etherscan’s contract interaction GUI. If you already have a position (i.e. if you’ve already minted tokens, these steps may not apply to you). The screenshots and example numbers used in this tutorial reflect the ETHBTC token, but all steps in this article apply just the same to any other expiring multi-party token (like the yCOMP). All you’ll need to get started is some DAI to use as collateral (50 DAI should be enough) and ETH to pay for transaction gas (0.05 ETH should be enough).
```
Comment (Tommy1231232) [inline docs/tutorials/mint-etherscan.md:6]: Nit pick but instead of article I would say section

File: docs/uma-tokenholders/adding-price-id.md [hunk 2]
```diff
@@ -71,11 +71,11 @@ To view the list of approved mainnet price identifiers, see below or run the `Su
...
-|PERL|  [UMIP-12](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-12.md)
-|DAI| 
-|USDC; USDT|  [UMIP-18](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-18.md)
-|rDAI|  [UMIP-17](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-17.md)
+|Collateral Currency| Final Fee | Link to UMIP|
+|:-------| :-----------| :-----------|
+|WETH| 1 |  [UMIP-10](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-10.md)
+|renBTC| 0.035 |  [UMIP-11](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-11.md)
+|PERL| 8600 | [UMIP-12](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-12.md)
+|DAI| 400 |
```
Comment (Tommy1231232) [inline docs/uma-tokenholders/adding-price-id.md:79]: Should the UMIP be the same as rDAI?

**Review by Tommy1231232 — COMMENTED** [review]
A few minor comments. Looks good!

I noticed the under Funding accounts, the Kovan test faucet link does not work: https://docs-git-tutorialsreorg.uma.vercel.app/tutorials/bots

-After each colon in the parameters sections if you want to make the letters capital that would good:
https://docs-git-tutorialsreorg.uma.vercel.app/tutorials/bot-param

-Link to collateral type like how it is for price identifier:
https://docs-git-tutorialsreorg.uma.vercel.app/tutorials/contract-deployment

File: docs/uma-tokenholders/adding-price-id.md (hunk 2 above)
Comment (smb2796) [reply docs/uma-tokenholders/adding-price-id.md:79]: Grandfathered in with the introduction of the whitelist. Adding the whitelist umip-8 link

**Comment by smb2796** [comment]
Thanks Tom! Comments addressed

**Pull request merged**